Makefile

```diff
@@ -0,0 +1,290 @@
+
+  ```
+  source $$(brew --prefix asdf)/libexec/asdf.sh
+  ```
+
+  You can do it by copy/paste of the following line
+  ```
+  echo -e "\n. $$(brew --prefix asdf)/libexec/asdf.sh" >> ~/.bash_profile
+  ```
+
+= Setting up direnv
+
+  Run following commands to install direnv plugin
+
+    ```
+    asdf plugin-add direnv
+    asdf install direnv latest
+    asdf global direnv latest
+    ```
+
+  Once you have done with above commands add following to the ~/.bash_profile
+
+    ```
+    # Hook direnv into your shell.
+    eval "$$(asdf exec direnv hook bash)"
+
+    # A shortcut for asdf managed direnv.
+    direnv() { asdf exec direnv "$$@"; }
+    ```
+
+    You can do it by copy/paste of the following
+
+    ```
+    cat << EOF >> ~/.bashrc
+    # Hook direnv into your shell.
+    eval "$$(asdf exec direnv hook bash)"
+
+    # A shortcut for asdf managed direnv.
+    direnv() { asdf exec direnv "$$@"; }
+    EOF
+    ```
+
+= Enabling direnv integration
+
+- After finishing the above two steps restart your shell.
+- Run following `asdf direnv setup --shell bash --version latest`
+- Restart your shell once again
+
+endef
+export BASH_DARWIN_HELP
+
+define ZSH_DARWIN_HELP
+
+There are two steps which need to be done to set up environment.
+
+  -  The asdf tool manager would need to be installed
+  -  The direnv plugin for asdf manager would need to be installed
+
+
+= Setting up asdf tool manager
+
+  You can install asdf from brew by following command.
+
+  ```
+  brew install asdf
+  ```
+
+  Once you finish installation add following line to ~/.zshrc
+  (or ${ZDOTDIR}/.zshrc if you use custom location).
+
+  ```
+  source $$(brew --prefix asdf)/libexec/asdf.sh
+  ```
+
+  You can do it by copy/paste of the following line
+  ```
+  echo -e "\n. $$(brew --prefix asdf)/libexec/asdf.sh" >> $${ZDOTDIR:-~}/.zshrc
+  ```
+
+= Setting up direnv
+
+  Run following commands to install direnv plugin
+
+    ```
+    asdf plugin-add direnv
+    asdf install direnv latest
+    asdf global direnv latest
+    ```
+
+  Once you have done with above commands add following to the ~/.zshrc
+  (or ${ZDOTDIR}/.zshrc if you use custom location).
+
+
+    ```
+    # Hook direnv into your shell.
+    eval "$$(asdf exec direnv hook zsh)"
+
+    # A shortcut for asdf managed direnv.
+    direnv() { asdf exec direnv "$$@"; }
+    ```
+
+    You can do it by copy/paste of the following
+
+    ```
+    cat << EOF >>  ~/.zshrc
+    # Hook direnv into your shell.
+    eval "$$(asdf exec direnv hook zsh)"
+
+    # A shortcut for asdf managed direnv.
+    direnv() { asdf exec direnv "$$@"; }
+    EOF
+    ```
+
+= Enabling direnv integration
+
+- After finishing the above two steps restart your shell.
+- Run following `asdf direnv setup --shell zsh --version latest`
+- Restart your shell once again
+
+endef
+export ZSH_DARWIN_HELP
+
+
+define CONTRIBUTE_SHELL_SETUP
+  We detected $(CURRENT_SHELL) running on $(CURRENT_OS). Unfortunatelly
```

Comment:
s/Unfortunatelly/Unfortunately/

Reply:
Fixed in 42827b4ee35ae4747a991264a25d5fc17b19236d